aws/table_aws_config_conformance_pack.go

```diff
@@ -0,0 +1,172 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/configservice"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsConfigConformancePack(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_config_conformance_pack",
+		Description: "AWS Config Conformance Pack",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("name"),
+			ShouldIgnoreError: isNotFoundError([]string{"NoSuchConformancePackException"}),
+			Hydrate:           getConfigConformancePack,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listConfigConformancePacks,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Name of the conformance pack.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ConformancePackName"),
+			},
+			{
+				Name:        "conformance_pack_id",
+				Description: "ID of the conformance pack.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ConformancePackId"),
+			},
+			{
+				Name:        "input_parameters",
+				Description: "A list of ConformancePackInputParameter objects.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ConformancePackInputParameters"),
+			},
+
+			{
+				Name:        "created_by_svc",
+				Description: "AWS service that created the conformance pack.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CreatedBy"),
+			},
+			{
+				Name:        "delivery_bucket",
+				Description: "Amazon S3 bucket where AWS Config stores conformance pack templates.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DeliveryS3Bucket"),
+			},
+			{
+				Name:        "delivery_bucket_prefix",
+				Description: "The prefix for the Amazon S3 delivery bucket",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DeliveryS3KeyPrefix"),
+			},
+			{
+				Name:        "last_update",
+				Description: "Last update to the conformance pack.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("LastUpdateRequestedTime"),
+			},
+			// Standard columns
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsConformancePackAkas,
+				Transform:   transform.FromValue(),
```

Comment:
So does this mean that it wouldn't need the hydrate parameter as well?

Reply:
Yep, that's correct ... the Hydrate is not needed since you can get the ARN directly from the existing field and then just transform it into an array using arnToAkas.

Reply:
This is a good example of a similar field https://github.com/turbot/steampipe-plugin-aws/blob/d9684b1adf979f1b17588cfdffec892a3fff1844/aws/table_aws_dynamodb_backup.go#L88